hetmech/matrix.py

```diff
@@ -66,3 +67,29 @@ def copy_array(matrix, copy=True, dtype=numpy.float64):
         mat_type = numpy.array
     matrix = mat_type(matrix, dtype=dtype, copy=copy)
     return matrix
+
+
+def permute_matrix(adjacency_matrix, directed=False, multiplier=10,
+                   excluded_pair_set=set(), seed=0):
+
```

Comment:
Let's add a docstring that this currently assumes boolean adjacency matrices... i.e. it doesn't deal with edge weigts